diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index e2891369fb49ee27eb139a210375bc3f1608440b..c481260f01d354a6cb0de80c70d0e241d22537d0 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -67,6 +67,7 @@ source_set("channel_info") {
     "//build:branding_buildflags",
     "//build:chromeos_buildflags",
   ]
+  deps += [ "//brave/common:channel_info" ]
   public_deps = [
     "//base",
     "//components/version_info",
@@ -153,6 +154,7 @@ static_library("common") {
   ]
 
   public_deps = [
+    "//brave/common",
     ":available_offline_content_mojom",
     ":buildflags",
     ":channel_info",
@@ -545,6 +547,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
